Sources/AuthFoundation/OAuth2/Authentication.swift

```diff
@@ -127,22 +130,29 @@ public struct StandardAuthenticationContext: AuthenticationContext {
     /// - Parameters:
     ///   - acrValues: Authentication Context Reference values to include with this sign-in.
     ///   - additionalParameters: Custom request parameters to be added to requests made for this sign-in.
-    public init(acrValues: [String]? = nil,
+    public init(acrValues: ClaimCollection<[String]?> = nil,
```

Comment:
This might be a Swift-ism I don't understand, but shouldn't this be

```swift
ClaimCollection<[String]>? = nil
```

Reply:
No, this is a swiftism related to property wrappers and the "ExpressibleByNilLiteral" protocol the ClaimCollection type implements.  This is functionally identical to writing:

```
public init(acrValues: ClaimCollection<[String]?> = ClaimCollection(nilLiteral: ())
```

Ultimately the ClaimCollection will never be nil, just the wrapped value it represents may be nil.